internal/cli/clients/client.go

```diff
@@ -1274,15 +1287,124 @@ spec:
 		Route   string
 		Org     string
 	}{
-		AppName: name,
+		AppName: appName,
 		Route:   route,
 		Org:     c.Config.Org,
 	})
+
 	if err != nil {
-		return "", errors.Wrap(err, "failed to render kube resource definition")
+		return errors.Wrap(err, "failed to render kube resource definition")
 	}
 
-	return
+	return nil
+}
+
+func (c *EpinioClient) renderService(filePath, appName string) error {
+	serviceTmpl, err := template.New("service").Parse(`
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    traefik.ingress.kubernetes.io/router.entrypoints: websecure
+    traefik.ingress.kubernetes.io/router.tls: "true"
+  labels:
+    app.kubernetes.io/component: application
+    app.kubernetes.io/managed-by: epinio
+    app.kubernetes.io/name: {{ .AppName }}
+    app.kubernetes.io/part-of: {{ .Org }}
+    kubernetes.io/ingress.class: traefik
+  name: {{ .AppName }}
+  namespace: {{ .Org }}
+spec:
+  ports:
+  - port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app.kubernetes.io/name: "{{ .AppName }}"
+  type: ClusterIP
+  `)
+	if err != nil {
+		return errors.Wrap(err, "failed to parse service template for app")
+	}
+
+	serviceFile, err := os.Create(filePath)
+	if err != nil {
+		return errors.Wrap(err, "failed to create file for application Service definition")
+	}
+	defer func() { err = serviceFile.Close() }()
```

Comment:
While this does make the linter happy, it does not check the err. I prefer 
```suggestion
	defer func() { _ = serviceFile.Close() }()
```

Or we disable the errchk linter?

Reply:
done